tests/regression/fn/fn_vertical_indirection.cpp

```diff
@@ -0,0 +1,68 @@
+/*
+ * GridTools
+ *
+ * Copyright (c) 2014-2021, ETH Zurich
+ * All rights reserved.
+ *
+ * Please, refer to the LICENSE file in the root directory.
+ * SPDX-License-Identifier: BSD-3-Clause
+ */
+
+#include <cstddef>
+#include <gtest/gtest.h>
+
+#include <gridtools/fn/cartesian.hpp>
+#include <gridtools/fn/unstructured.hpp>
+
+#include <fn_select.hpp>
+#include <test_environment.hpp>
+
+namespace {
+    using namespace gridtools;
+    using namespace fn;
+    using namespace literals;
+
+    struct derivative_stencil {
+        auto operator()() const {
```

Comment:
```suggestion
        GT_FUNCTION constexpr auto operator()() const {
```
`GT_FUNCTION` is needed for NVCC (constexpr might be enough in the future).

Reply:
Fixed it.

Reply:
Is this test actually compiled for CUDA as well? If not, should we make it?

Reply:
It should be compiled with CUDA already. You can also double-check by looking at the console output on Jenkins.

Reply:
Yeah, double checked it and it runs on CUDA proper.